python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -181,14 +181,17 @@ class ValidationHandler(TrainBegin, BatchEnd, EpochEnd):
         Priority level of the ValidationHandler. Priority level is sorted in
         ascending order. The lower the number is, the higher priority level the
         handler is.
+    event_handlers : EventHandler or list of EventHandlers
```

Comment:
Could you add more detailed doc? Something like " EventHandler or list of EventHandlers to be used by the `eval_fn`". Otherwise, it will be confusing for users.

Reply:
doc string added